contrib/pg_tde/documentation/docs/yum.md

```diff
@@ -4,6 +4,24 @@ This tutorial shows how to install `pg_tde` with [Percona Distribution for Postg
 
 Check the [list of supported platforms](install.md#__tabbed_1_1).
 
+## Memory limits for `pg_tde` keys
+
+A memory lock (`mlock`) is the maximum amount of RAM a process can occupy. This value differs between operating systems. You can check the current setting with this command:
+
+```
+ulimit -a 
+```
+
+For example, on Rocky Linux 8, the default `mlock` limit is 64 Kb. An internal `pg_tde` key size is 40 bytes. Thus, the `mlock` limit is sufficient for about 1600 internal keys. When this limit is reached, `pg_tde` cannot lock memory for more keys and can fail with the error. 
```

Comment:
this issue was arm specific, but maybe it's even smaller there? I don't remember that part

Reply:
The keys are per backend so if you have 100 backends that is only 16 keys per backend. You need way more than 64 kb for even a small setup.

Reply:
Do I need to explicitly mention this is for ARM?

Reply:
@jeltz what is meant by a backend? Any recommendations then?

Reply:
@jeltz but limits are per-process

Reply:
But the issue was about Rocky Linux specifically

Reply:
> @jeltz what is meant by a backend? Any recommendations then?

Roughly number of connections + parallel workers + background workers but in practice you can look at it as the number of connections since that one is usually much larger than the others. But now I also realized the limit per process and not per user so it does not matter. Your original math was correct. :D

Reply:
Thanks. So are you OK with the wording? I need your approve for the merge:)